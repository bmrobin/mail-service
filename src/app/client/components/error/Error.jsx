import React from 'react';
import * as propTypes from 'prop-types';
import './error.css';

export default function Error(props) {
  return (
    <p className="error">{props.message}</p>
  );
}

Error.propTypes = {
  message: propTypes.string.isRequired
};
