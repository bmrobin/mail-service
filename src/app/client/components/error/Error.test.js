import React from 'react';
import renderer from 'react-test-renderer';
import Error from './Error';

describe("Error Component", () => {
  
  test("should render component", () => {
    const component = renderer.create(<Error message={"an error message"} />);
    expect(component).toMatchSnapshot();
  });

});
