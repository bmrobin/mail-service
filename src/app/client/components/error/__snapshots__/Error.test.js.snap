// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component should render component 1`] = `
<p
  className="error"
>
  an error message
</p>
`;
